database/languages/golang.json

```diff
@@ -1,3 +1,9 @@
 [
-    
-]
\ No newline at end of file
+  {
+    "name": "Hitesh Choudhary",
+    "description": "This Playlist of Hitesh Choudhary contains all the cocepts of Golang along with two mini projects",
```

Comment:
```suggestion
    "description": "This playlist contains all the concepts of Golang along with two mini projects",
```

Reply:
hey @CBID2 i have applied the suggested changes